packages/react/src/data-table/DataTable.css.ts

```diff
@@ -0,0 +1,61 @@
+import { createVar, recipe, style } from "../vanilla-extract";
+
+export const columnWidth = createVar();
+export const cellOffset = createVar();
+
+export const tableHeader = recipe({
+  base: [
+    {
+      bg: "white",
+    },
+    style({
+      position: "sticky",
+      top: 0,
+      zIndex: 2,
+    }),
+  ],
+});
+
+export const tableHead = recipe({
+  base: [
+    {
+      bg: "white",
+    },
+    style({
+      minWidth: columnWidth,
+    }),
+  ],
+  variants: {
+    pinned: {
+      false: style({}),
+      left: style({
+        left: cellOffset,
+        position: "sticky",
+        zIndex: 2,
+      }),
+      right: style({}),
+    },
+  },
+});
+
+export const tableCell = recipe({
+  base: [
+    {
+      bg: "white",
+    },
+    style({
+      minWidth: columnWidth,
+    }),
+  ],
+  variants: {
+    pinned: {
+      false: style({}),
```

Comment:
let's replace with just `{}` so we don't create blank class names needlessly
```suggestion
      false: {},
```